packages/url_launcher/ios/Classes/UrlLauncherPlugin.m

```diff
@@ -118,11 +127,15 @@ - (void)launchURL:(NSString *)urlString result:(FlutterResult)result {
 
 - (void)launchURLInVC:(NSString *)urlString result:(FlutterResult)result {
   NSURL *url = [NSURL URLWithString:urlString];
-
-  SFSafariViewController *safari = [[SFSafariViewController alloc] initWithURL:url];
   _currentSession = [[FLTUrlLaunchSession alloc] initWithUrl:url withFlutterResult:result];
-  safari.delegate = _currentSession;
-  [_viewController presentViewController:safari animated:YES completion:nil];
```

Comment:
We might have a small memory leak here. We allocate _currentSession but we never deallocate it. Next launch URL call will cause the previous session to be garbage collected but it would be good to add a completion handler here to delete the session.